tests/integration/cloud/rest_contracts/test_checkpoints.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import pathlib
+from typing import TYPE_CHECKING, Callable, Final
+
+import pact
+import pytest
+
+from tests.integration.cloud.rest_contracts.conftest import (
+    EXISTING_ORGANIZATION_ID,
+    ContractInteraction,
+)
+
+if TYPE_CHECKING:
+    from tests.integration.cloud.rest_contracts.conftest import PactBody
+
+
+NON_EXISTENT_CHECKPOINT_ID: Final[str] = "6ed9a340-8469-4ee2-a300-ffbe5d09b49d"
+
+EXISTING_CHECKPOINT_ID: Final[str] = "051a1f4d-6276-484a-81e5-3df4fadd5154"
```

Comment:
Where did you find this Checkpoint? I don't see it in the Checkpoints table of the Mercury seed data.

Reply:
I thought I got it from my local db but I guess not. Just updated with EXISTING_CHECKPOINT_ID: Final[str] = "4d734009-2c50-4222-bd71-9660f5b05fff" here and in my other two PRs.